testing/functional/tests/cypress.json

```diff
@@ -0,0 +1,21 @@
+{
+    "baseUrl":"https://test-aest-sims.apps.silver.devops.gov.bc.ca/student",
```

Comment:
Is it possible to move the baseURL in some configuration or like a variable outside? So that we can test multiple URL's, If I understood it wrong please clarify me. Thanks

Reply:
Yes multiple URLs would be possible. Will make this change next week after i complete one section of all cases from student perspective.